faster-map.opam

```diff
@@ -1,9 +1,14 @@
-opam-version: "1.2"
+opam-version: "2.0"
+synopsis: "A tail-recursive list map with good performance for all list sizes. Not actually written in assembly"
+authors: "<Anton Bachin> antonbachin@yahoo.com"
+maintainer: "<Anton Bachin> antonbachin@yahoo.com"
+homepage: "https://github.com/aantron/faster-map"
+bug-reports: "https://github.com/aantron/faster-map/issues"
 
 depends: [
   "base"
   "batteries"
   "containers"
   "core_bench"
-  "jbuilder" {build & >= "1.0+beta13"}
+  "dune" {build & >= "1.0"}
```

Comment:
`dune` should be a non-build dependency. See https://github.com/ocaml/opam-repository/pull/14266.